egs/aishell/s10/chain/tdnnf_layer.py

```diff
@@ -194,13 +187,13 @@ def forward(self, x):
 
         # save it for skip connection
         input_x = x
-
+        logging.debug(f"input_x shape is {input_x.shape}")
```

Comment:
I think f-style string is only available in python 3.6.

Could you please use `"".format()` instead.

Reply:
thanks, looks good to me, merging.